src/makefun/main.py

```diff
@@ -106,6 +107,7 @@ def create_function(func_signature,             # type: Union[str, Signature]
                     add_impl=True,              # type: bool
                     doc=None,                   # type: str
                     qualname=None,              # type: str
+                    co_name=None,               # type: str
```

Comment:
Could you please update the body of the docstring to explain precisely the new behaviour ?
It is indeed hard to understand under which conditions lambdas are created vs. standard functions, as well as why users would like to change the new parameter.